src/key_info_managers/on_disk_manager/mod.rs

```diff
@@ -303,48 +303,75 @@ impl OnDiskKeyInfoManager {
 }
 
 impl ManageKeyInfo for OnDiskKeyInfoManager {
-    fn get(&self, key_triple: &KeyTriple) -> Result<Option<&KeyInfo>, String> {
+    fn get(&self, key_identity: &KeyIdentity) -> Result<Option<&KeyInfo>, String> {
+        let key_triple = KeyTriple::try_from(key_identity.clone()).map_err(|_e| {
+            error!("Error creating KeyTriple from KeyIdentity");
+            "Error creating KeyTriple from KeyIdentity"
```

Comment:
Seems a bit superfluous, I think the KIM client then goes ahead and prints what it gets from this function as an `error!` as well, it might be better to just return the `&str` (?)